### DISCUSSION TITLE
Option to enable or disable Matrix items per site

### DISCUSSION THREAD

**Discussion by shoored** [craftcms/cms#15093] [Ideas]
While trying things out with Matrix fields in Craft 5 and the different View Modes I came across this "issue". I'll try to explain myself. 

When in "cards mode" I can double click a Matrix field item and then disable this entry in the slideout. This item will then have a disabled status indicator. This works wonderfully in a multi-site setup where I want to have an identitical Matrix structure for the Dutch and English site for example, translate some content and even _disable_ a particular Matrix item for a site while keeping it active for the others.

When in "inline-editable blocks mode" that same block can be disabled for a particular site and it won't render on the front-end but it's not clear in the CP that this is the case. You would have to open the Matrix entry in a new tab via the three dots menu to see this type of status and change it.

Now when I disable that Matrix item the old way, this item is disabled in all sites and you get the red disabled status indicator. 

On a Craft 4 site, the client wanted the option to have Matrix blocks only enabled for a particular site. We implemented this with lightswitch fields in all block types. It seems in Craft 5 this can be done more natively with the "enabled / disabled for site" but it's not working exactly the same as disabling a Matrix item. Perhaps because that is not the way to go, but it kinda feels like a loophole if you know what I mean?

It would be nice if we could "disable" Matrix items per site but keep them "enabled" for others, but not just in "cards mode".

Also, I came across [this discussion](https://github.com/craftcms/cms/discussions/15076) but did that seems to talk about a higher level status.

![cards-mode](https://github.com/craftcms/cms/assets/26165948/cc00a38f-4ec9-4740-85af-86405b1e1470)
![inline-editable-mode](https://github.com/craftcms/cms/assets/26165948/9cc04d4b-c9a1-4397-9be7-e8ebf976516e)

**Comment by yingban** [comment]
@shoored 

I think I would have more or less the same feature request.
Our clients build entries content with matrix blocks, which are propagated in each language (blocks are the same, content is translated), but it can happen that a translation is not ready in one language, but the block cannot be disable in the not ready site.

**Comment by maxstrebel** [comment]
I have the same problem and thought that this worked at some point 😅 is it possible to translate the visibility of a matrix block in some way?
